google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -234,6 +243,13 @@ public static class Builder {
     private long initialWaitForSessionTimeoutMillis = 30_000L;
     private ActionOnSessionNotFound actionOnSessionNotFound = ActionOnSessionNotFound.RETRY;
     private ActionOnSessionLeak actionOnSessionLeak = ActionOnSessionLeak.WARN;
+    /**
+     * Capture the call stack of the thread that checked out a session of the pool. Can be disabled
+     * by customers who do not want this, for example if their monitoring systems think this is an
+     * exception that they need to log.
```

Comment:
IIUC, customers would like to disable `trackStackTraceOfSessionCheckout` if they do not want our library to pre-create exception during session checkout (but create it when there is an actual session leak) so that their monitoring systems do not log this creation (but log it only if session is actually leaked). Shall we reword "for example if their monitoring systems think this is an exception that they need to log." to include this detail?

Reply:
Reworded